packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -417,7 +423,13 @@ module.exports = class ImportEntries {
                           }
                           // TODO: if status code: 422, check the reason
                           // 429 for rate limit
-                          log(this.config, `Failed to create an entry ${eUid} ${formatError(error)}`, 'error');
+                          log(
+                            this.config,
+                            `Failed to create an entry ${eUid} ${formatError(error)}. Title of the failed entry: ${
+                              entries[eUid].title
```

Comment:
@cs-raj Please use the quotes for the entity name for better readability.

Reply:
Done